### PR TITLE
Custom CSS & Clock improvements

### PR DESCRIPTION
**Pull request by rob1998**
The stylesheet is already loaded in <head> == bad for performance
Loading it again overwrites all custom css, that doesn't include !important

Second commit adds a working javascript clock, both analog and digital displaying server time

### CODE REVIEW

**Comment by seanvree** [comment]
@rob1998  can you push this to alpha branch?

**Comment by seanvree** [comment]
Hey @rob1998  good catch on the CSS thing. 

One question about the clock update.  So, if I'm understanding this code right - the UI will load server time onload? Or does it?  I see it gets the timezone, but how is it actually getting the server's time? I'm assuming it just uses browser time with the server's timezone.   Before it was calling timestamp, so it was getting real servertime on the loop - which may have been a bit of an overkill.  Just wondering so I understand.

**Comment by rob1998** [comment]
@seanvree The analog clock.js does all the work actually. I noticed this while building another solution. The servertime is stored in a variable in when the pageloads, then the clock.js file creates a date object (and stores it in a variable called date) using that time and adds a second every 1000 milliseconds. I use the same date variable to update the digital clock, this variable keeps getting updated for the analog clock, so it's automatically also updating for the digital clock.
the rftime variable only controls how often the digital clock value is set to the value of the date variable.
As far as I can see the timezone variable isn't used atm (although it's present in the clock.js, but not in use). Could be used to display to the user which timezone is being displayed.
So in summary: it get's the servertime once, then with javascript code it adds a second to that time every second.

Instead of my previous solution to put the date number in front of the month, you could also try switching the day en month in the timeString variable. Cause I think it also controls the order, not sure though

**Comment by seanvree** [comment]
@rob1998  thanks for the explanation.  Good work my brother. 

One small request - is there a way to display the timezone to the right of the seconds ONLY when in 24-hour format?

**Comment by rob1998** [comment]
@seanvree yeah should be possible, can you push all your changes and tell me which branch I should test it on?

**Comment by seanvree** [comment]
I have not pulled the new clock in from the PR yet.  However, I have significantly updated the ALPHA branch this morning.

**Comment by rob1998** [comment]
@seanvree I think this one can be closed right?

**Comment by seanvree** [comment]
@rob1998  yup. added manually.
